jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/java/JavaLambdaLineMarkerTest.kt

```diff
@@ -66,6 +67,11 @@ class JavaLambdaLineMarkerTest {
         )
     }
 
+    @After
+    fun tearDown() {
+        MockProjectAccountSettingsManager.getInstance(projectRule.project).reset()
```

Comment:
This is added here because `software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaLineMarkerTest#noCredentialsLeadsToNoMarkerIfNoOtherCriteriaPasses` clears out credentials.